### ISSUE TITLE
News listing: image should be clickable

### ISSUE THREAD

**Issue by jone** [4teamwork/ftw.news#42]
News listing improvements:

- The image should be clickable and lead the user to the details view.
- A link `Read more...` should be added below the text, also pointing to the details view.

![bildschirmfoto 2016-02-12 um 15 58 24](https://cloud.githubusercontent.com/assets/7469/13010350/ba385f30-d1a1-11e5-9a90-2c04aa63a286.png)

/cc @bbuehlmann

**Comment by mbaechtold** [comment]
@bierik and I have analyzed the current situation and we have noticed that the news portlet and the news listing view both look and behave the same way: 
- the whole news item row is linked (including the image, title and description).
- the news item changes the style on hover so it becomes more obvious to the user that he can click the news item. 
- the previous point is the reason why there is no `Read more...` button. It's not needed.

The news listing block on the other hand behaves differently: 
- the whole news item is *not* linked.
- only the title and the `Read more...` button ist linked.

We suggest to make all these listings look and behave the same way. We would therefor update the news listing block, which would result in the following:

**News listing view (unchanged):**

<img width="1440" alt="screenshot news 1" src="https://cloud.githubusercontent.com/assets/28220/14170781/bf9c40be-f72f-11e5-9592-f60f01ddd520.png">

**News portlet (unchanged):**

<img width="1440" alt="screenshot news 3" src="https://cloud.githubusercontent.com/assets/28220/14170788/c768bc14-f72f-11e5-95f8-223dfd4a903f.png">

**News listing block (needs to changed):**

<img width="1440" alt="screenshot news 2" src="https://cloud.githubusercontent.com/assets/28220/14170785/c3b8ff5c-f72f-11e5-8fae-5abbf9321ded.png">
